packages/neo4j/src/index.ts

```diff
@@ -0,0 +1,97 @@
+import neo4j from "neo4j-driver"
+import type { Adapter } from "next-auth/adapters"
+
+import {
+  createUser,
+  getUser,
+  getUserByEmail,
+  getUserByAccount,
+  updateUser,
+  deleteUser,
+} from "./user"
+
+import { linkAccount, unlinkAccount } from "./account"
+
+import {
+  createSession,
+  getSessionAndUser,
+  updateSession,
+  deleteSession,
+} from "./session"
+
+import {
+  createVerificationToken,
+  useVerificationToken,
+} from "./verificationToken"
+
+export function Neo4jAdapter(neo4jSession: typeof neo4j.Session): Adapter {
+  return {
+    async createUser(data) {
+      return await createUser(neo4jSession, data)
+    },
+
+    async getUser(id) {
+      return await getUser(neo4jSession, id)
+    },
+
+    async getUserByEmail(email) {
+      return await getUserByEmail(neo4jSession, email)
+    },
+
+    // TODO:
+    // Question: Was providerId and provider accountID now single parameter.
+    // providerAccountId: Pick<Account, "provider" | "providerAccountId">
+    // Will this always be of type providerAccountId ?
+    // If not, when would it be type provider ?
+
+    async getUserByAccount(provider_providerAccountId) {
+      return await getUserByAccount(neo4jSession, provider_providerAccountId)
+    },
+
+    async updateUser(data) {
+      return await updateUser(neo4jSession, data)
+    },
+
+    async deleteUser(id) {
+      return await deleteUser(neo4jSession, id)
+    },
+
+    async linkAccount(data) {
+      return await linkAccount(neo4jSession, data)
+    },
+
+    // TODO:
+    // Same question as with getUserByAccount
+
+    async unlinkAccount(provider_providerAccountId) {
+      return await unlinkAccount(neo4jSession, provider_providerAccountId)
+    },
+
+    async createSession(data) {
+      return await createSession(neo4jSession, data)
+    },
+
+    async getSessionAndUser(sessionToken) {
+      return await getSessionAndUser(neo4jSession, sessionToken)
+    },
+
+    // TODO:
+    // Question: force parameter is no longer used?
```

Comment:
No. Most of the logic is moved to the core, but the `force` param wasn't passed anywhere, so technically it has **never** been used... :shrug: